akka-typed/src/test/scala/akka/typed/ActorContextSpec.scala

```diff
@@ -73,6 +73,36 @@ object ActorContextSpec {
   final case class GetAdapter(replyTo: ActorRef[Adapter], name: String = "") extends Command
   final case class Adapter(a: ActorRef[Command]) extends Event
 
+  val foo1: Actor.Stateless[Command] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+    case (_, Throw(ex)) ⇒
+      throw ex
+    case _ ⇒ ()
+  }
+
+  val foo2: Behavior[Command] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+    case (_, Throw(ex)) ⇒
+      throw ex
+    case _ ⇒ ()
+  }
+
+  val foo3: Behavior[Ping] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+  }
+
+  // TODO missing parameter type for expanded function The argument types of an anonymous function must be fully known.
+  //      (SLS 8.5) Expected type was: Function2[akka.typed.scaladsl.ActorContext[akka.typed.ActorContextSpec.Ping], akka.typed.ActorContextSpec.Ping, ?]
+  /*
+  val foo4 = Actor.Stateless[Ping] {
```

Comment:
ok, always `(ctx, msg) =>` and then match on `msg` separately

Yes, I think it should be changed to `Any`, and then we can also support `Unhandled` from `Stateless`:

```scala
  val foo5 = Actor.Stateful[Command] { (ctx, msg) =>
    msg match {
      case Ping(replyTo) ⇒
        replyTo ! Pong2
        Actor.Same
      case _ => Actor.Unhandled
    }
  }

  val foo6 = Actor.Stateless[Command] { (ctx, msg) =>
    msg match {
      case Ping(replyTo) ⇒
        replyTo ! Pong2
      case _ => Actor.Unhandled
    }
  }
```